### ISSUE TITLE
anatomical overlaps

### ISSUE THREAD

**Issue by patrick-lloyd-ray** [patrick-lloyd-ray/AIO-Aug2020#3]
If we wanted to assert that some specimen overlaps two anatomical regions, how would we do this?

**Comment by jimmymathews** [comment]
If you mean in reference to the common_model.csv stuff, I think the answer is: By providing two different instance property assertions.

**Comment by jimmymathews** [comment]
There are two cases for that; either they overlap specimen S for a common reason (there exists some z which is both part of x and part of y, and z is part of S), or they overlap for two different reasons (there exists some z1 which is part of x and z2 which is part of y, and z1 and z2 are both part of S). I'm not sure if distinguishing these is what you had in mind.

**Comment by patrick-lloyd-ray** [comment]
Yes, that is one case: say there is some anatomical schema (A) and another (B). According to A, there are certain subregions [a1, a2, a3,...,an]. Likewise for B, there are [b1, b2, b3,...,bn]. The first case is where some S overlaps <a1, a2>, or <b2, b3>. This seems like the case you have above, I think (but correct me if I'm wrong).

Another case might be an assertion to the effect that some S is in <a3> and <b1>, but <a3> and <b1> do not share all parts. Perhaps because (A) is grounded in functional region or connectivity, but (B) is grounded in surface features (using gyri or sulci). Further, I might want to make an assertion about the schemas comparatively -- or to define one in terms of the other. I don't know if I would want to say it "overlaps" here, but I would certainly like to make assertions of some sort to enable reasoning over those assertions. Maybe it's the same problem occurring at the level of schema and the solution would be the same, but simply require more complexity in expression. 

Certainly, if I know to a very fine granularity (cell soma perhaps), then this is moot. But perhaps I just know that a cell is in one region according to one schema and another according to another schema, but those regions are broad (let's say). Maybe I would use an intersection or something like that -- but now it's an intersection of two different schema regions. We'd need something to map one schema to another, I think. Maybe there is an obvious solution here or the problem is ill-conceived.

This is more of an interoperability problem that probably doesn't need attention now. [A similar but distinct problem on the Human Brain Atlas: https://community.brain-map.org/t/atlas-anatomy-terminology-and-other-anatomical-notations/204]

**Comment by jimmymathews** [comment]
Yes, since the context is being able to include assertions that specimens belong to regions, I think the question of overlapping regions is a little out of the current scope.
Using "overlap" in this context earlier was a mistake on my part, permitted due to an unreasonably material interpretation of "region". I think we decided it will have to become something like:

anatomical registration *has target* anatomical atlas part
anatomical registration *has source* specimen
anatomical registration *performed by* person
